jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -55,8 +55,8 @@
 import static java.lang.Integer.parseInt;
 import static java.lang.String.format;
 import static java.util.Arrays.asList;
-import static org.apache.commons.lang3.StringUtils.isBlank;
-import static org.apache.commons.lang3.StringUtils.isEmpty;
+import static org.apache.commons.lang3.StringUtils.*;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
```

Comment:
We don't need this import since we have import static org.apache.commons.lang3.StringUtils.*;